spyder/widgets/mixins.py

```diff
@@ -1362,7 +1362,7 @@ def _enter_array(self, inline):
 class TracebackLinksMixin(object):
     """ """
     QT_CLASS = None
-    go_to_error = None
+    sig_go_to_error_requested = None

```

Comment:
Please add a comment above this signal to describe what it does.

Reply:
Fixed